src/dex-ui/pages/dao/DAOProposals/Forms/DAOTokenTransferDetailsForm.tsx

```diff
@@ -119,36 +143,39 @@ export function DAOTokenTransferDetailsForm() {
         isInvalid={Boolean(errors?.recipientAccountId)}
         errorMessage={errors?.recipientAccountId && errors?.recipientAccountId?.message}
       />
-      <FormInput<"tokenId">
-        inputProps={{
-          id: "tokenId",
-          label: "Asset",
-          type: "text",
-          placeholder: "Enter a token ID",
-          register: {
-            ...register("tokenId", {
-              required: { value: true, message: "A token ID is required." },
-            }),
-          },
-          value: state?.tokenId,
-        }}
-        isInvalid={Boolean(errors?.tokenId)}
-        errorMessage={errors?.tokenId && errors?.tokenId?.message}
-      />
-      {/*
-       *  TODO: Replace Assets Input with Dropdown
-       *  <FormDropdown
-       *   label="Assets"
-       *   placeholder="Select an asset"
-       *   data={assetDropdownOptions}
-       *   isInvalid={Boolean(errors?.tokenId)}
-       *   errorMessage={errors?.tokenId && errors?.tokenId?.message}
-       *   register={register("tokenId", {
-       *     required: { value: true, message: "A token is required." },
-       *     onChange: (e: ChangeEvent<HTMLSelectElement>) => setValue("tokenId", e.target.value),
-       *   })}
-       * />
-       */}
+      {daoType === DAOType.MultiSig ? (
```

Comment:
`DAOType.MultiSig` is equal to "MultiSig". daoType is "multisig" when the DAO is a MultiSig DAO. Therefore, the dropdown is never shown.

Reply:
You are right. Made the changes are per your suggestions.